.github/workflows/android.yml

```diff
@@ -62,6 +62,10 @@ jobs:
           git config --global credential.helper 'store --file /tmp/git-credentials'
           echo 'https://${{ github.token }}@github.com' > /tmp/git-credentials
 
+      - name: Support Longpaths
```

Comment:
ultranit: Mention "Git" in the name here, just so it makes more sense when someone sees this out of context.

e.g. "Support Longpaths" -> "Enable Git Longpaths Support"

Reply:
Done.